### PR TITLE
Add user guides

### PR DESCRIPTION
**Pull request by gongjason**
* Added a few docs to the user guide section
* moved `User Guides` above `References` as it seems more important

### CODE REVIEW

File: docs/docs/user-guides/add-devices.md [hunk 1]
```diff
@@ -6,5 +6,12 @@ parent: User Guides
...
 
 Add a device through the Web UI by clicking the "Add Device" button under
-`/devices` or `/users`. Once the device profile is created, you can send the
-WireGuard configuration file to the user.
+`/devices` or `/users`. 
+
+![add device](https://user-images.githubusercontent.com/52545545/152582102-7bde379a-57d5-4f35-9b4e-4d02c32241bc.png)
+
+Once the device profile is created, you can send the WireGuard configuration file to the user by:
+* **Shareable Link**: Generates a time limited link to the device config file that can be.
```
Comment (jamilbk) [inline docs/docs/user-guides/add-devices.md]: > that can be

Sentence chopped off?

**Review by jamilbk — COMMENTED** [review]
One comment re: cutoff sentence

**Review by jamilbk — CHANGES_REQUESTED** [review]
https://github.com/firezone/firezone/pull/439/files#r799927053

File: docs/docs/user-guides/add-devices.md (hunk 1 above)
Comment (gongjason) [reply docs/docs/user-guides/add-devices.md]: fixed in new commit

**Pull request merged**